onnx/defs/object_detection/defs.cc

```diff
@@ -124,7 +124,7 @@ ONNX_OPERATOR_SET_SCHEMA(
           updateOutputShape(ctx, 0, {num_rois, C, ht, width});
         }));
 
-static const char* NonMaxSuppression_doc = R"DOC(
+static const char* NonMaxSuppression_ver11_doc = R"DOC(
```

Comment:
Is this necessary? The first renaming looks good but this may be not necessary.

Reply:
You are right, it was usefull in the version I started with, but not anymore.
I think it unifies the naming though! I can remove it if you think it is not worth it.